src/groups/mqb/mqbs/mqbs_filebackedstorage.cpp

```diff
@@ -389,6 +389,10 @@ FileBackedStorage::confirm(const bmqt::MessageGUID& msgGUID,
     RecordHandlesArray& handles = it->second.d_array;
     BSLS_ASSERT_SAFE(!handles.empty());
 
+    if (0 == --it->second.d_refCount) {
```

Comment:
Let's add a comment explicitly stating that we don't record the last confirm here for optimization